src/server/jsonrpc.rs

```diff
@@ -15,12 +15,17 @@ use warp::{Filter, Rejection, Reply};
 ///
 /// # Examples
 /// ```no_run
-/// use agent::node::config::JsonRPCServerConfig;
+/// use agent::config::Server as JsonRPCServerConfig;
 /// use agent::node::jsonrpc::JsonRPCServer;
 ///
+/// use std::net::SocketAddr;
+///
 /// #[tokio::main]
 /// async fn main() {
-///     let n = JsonRPCServer::new(JsonRPCServerConfig::default());
+///     let config = JsonRPCServerConfig {
+///         json_rpc_address: SocketAddr::from_str(server_json_rpc_addr).unwrap()
+///     }
+///     let n = JsonRPCServer::new(config);
```

Comment:
Wouldn't it make more sense to share an example here that reads the config for the json rpc server from the config file instead of manually constructing the server config from an address variable? (is should be just a few more lines of code in the example).

Reply:
works too!